ttyplot.c

```diff
@@ -54,10 +54,7 @@
 #define T_LLCR ACS_LLCORNER
 #endif
 
-static sigset_t block_sigset;
-static sigset_t empty_sigset;
-static volatile sig_atomic_t sigint_pending = 0;
-static volatile sig_atomic_t sigwinch_pending = 0;
+static int signals[2];  // pipe for sending signals to the main loop
```

Comment:
Just an idea: We could controduce two constants for read end (value 0) and write end (value 1) and use those in the code, to be further away from "why 0 here, what's the first one of the two again?".

Reply:
@edgar-bonet PS: maybe `signal_fds` or `signalfds` would be a better name.

Reply:
Indeed, it's kind of a magic number. I made it clearer by using two separate variables: `signal_read_fd` and `signal_write_fd`.

Reply:
@edgar-bonet definitely more clear, thanks! :+1: